map/api_mark_point.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "map/user_mark.hpp"
+#include "map/styled_point.hpp"
+
+class ApiMarkPoint : public StyledPoint
```

Comment:
Because a Bookmark is a StyledPoint but not an ApiMarkPoint. StyledPoint is the common ABC of both.